ui_framework/doc_site/src/views/toggle_button/toggle_button_example.js

```diff
@@ -24,7 +24,9 @@ export default props => (
       }]}
     >
       <GuideText>
-        You can use this button to reveal and hide content.
+        You can use this button to reveal and hide content. For a complete example
+        on how to make an collapsable panel proper accessible, read
+        the <a href="#/collapsebutton">CollapseButton</a> documentation.
```

Comment:
Instead of using `<a>` directly here, let's use react-router's Link:

```jsx
import {
  Link,
} from 'react-router';

<Link
  className="guideLink"
  to="collapsebutton"
>
  CollapseButton
</Link>
```